components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/DAO/GroupDAO.java

```diff
@@ -79,6 +79,40 @@
         return groups;
     }
 
+    /**
+     * Lists the groups that belong to a tenant by its tenant id that are created from SCIM
```

Comment:
Shall we add a `.` at the end of the line?

Reply:
Fixed with https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/pull/252/commits/98c7dd4b02908f8feb007d39d8f7fca4a7abae6d